kahuna/public/js/mixpanel/mixpanel.js

```diff
@@ -47,6 +46,6 @@ mp.factory('mixpanel', ['$window', 'mixpanelEnabled', function($window, mixpanel
     return {
         init: init,
         track: track,
-        isEnabled: () => mixpanelEnabled
+        isEnabled: () => initialised
```

Comment:
ah!

Feels better to have that check in [here](https://github.com/guardian/grid/blob/sc-mixpanel-token-in-api/kahuna/public/js/mixpanel/mixpanel.js#L38) so that the state is contained. Though that's a separate issue.